apps/wallet/src/ui/app/components/address-input/index.tsx

```diff
@@ -49,33 +49,35 @@ export function AddressInput({
         <>
             <div
                 className={cx(
-                    'flex h-11 py-1 w-full px-3 pr-0 items-center rounded-2lg bg-white border border-solid box-border focus-within:border-steel transition-all overflow-hidden',
+                    'flex h-max w-full rounded-2lg bg-white border border-solid box-border focus-within:border-steel transition-all overflow-hidden',
                     meta.touched && meta.error
                         ? 'border-issue'
                         : 'border-gray-45'
                 )}
             >
-                <TextareaAutosize
-                    maxRows={2}
-                    minRows={1}
-                    disabled={disabled}
-                    placeholder={placeholder}
-                    value={formattedValue}
-                    onChange={handleOnChange}
-                    onBlur={field.onBlur}
-                    className={cx(
-                        'w-full text-bodySmall leading-100 font-medium font-mono bg-white placeholder:text-steel-dark placeholder:font-normal placeholder:font-mono border-none resize-none',
-                        meta.touched && meta.error
-                            ? 'text-issue'
-                            : 'text-gray-90'
-                    )}
-                    name={name}
-                />
+                <div className="min-h-[42px] w-full flex items-center pl-3 py-1 pr-0 ">
```

Comment:
```suggestion
                <div className="min-h-[42px] w-full flex items-center pl-3 py-1 pr-0">
```

super nit haha

Reply:
I think we can just remove the `pr-0` class